matchms/Metadata.py

```diff
@@ -144,6 +144,26 @@ def items(self):
         """
         return self._data.items()
 
+    def to_dict(self, export_style: str = "matchms"):
+        """Returns a regular Python dictionary containing the metadata entries.
+
+        Parameters
+        ----------
+        export_style:
+            Specifies the naming style of the metadata fields.
+            Default is "matchms".
+        """
+        if export_style == "matchms":
+            return dict(self._data)
+        key_conversions = load_export_key_conversions(export_style=export_style)
+        keep_list = [x for x in self._data.keys() if x in key_conversions]
```

Comment:
This should actually have been checking if it is in the values, not the keys, as the keys are actually the keywords from matchms, not the style to export to